tools/creds/slack_users_map.yaml

```diff
@@ -0,0 +1,110 @@
+---
+anli: UB52DHXRR
+pruan: UBET0LUR3
+weinliu: UBGCU8SP9
+schoudha: UBJ7G20VB
+xtian: UBN967S59
+zhsun: UBWDDBW82
+wjiang: UBZUY3821
+dyan: UC28MR05B
+mifiedle: UC96WJCG2
+qitang: UCG1RMLJ0
+yapei: UCHAHGZQ9
+scheng: UCJ5CM18R
+juzhao: UCQUV73KJ
+wsun: UCZEVCSDU
+jhou: UD0F5G2GZ
+wabouham: UD2E0UZNF
+schituku: UD3T9Q2AJ
+jiazha: UD76XG8QM
+yasun: UD955LPJL
+geliu: UD955RU7J
+hongli: UD95K26TA
+jialiu: UD991138R
+yufchang: UD99NJ6NM
+lxia: UD9AYKQV7
+gpei: UD9MSAT28
+xiuwang: UD9NAEFPE
+wmeng: UD9NVFMA8
+wewang: UD9SG65V1
+yanpzhan: UD9T75SFN
+tzhou: UD9V7FBEW
+xxia: UDA3HA1GV
+zzhao: UDA3HP6ER
+jiajliu: UDAUE6EDV
+chezhang: UDAUK10RM
+yinzhou: UDB1X5DCP
+piqin: UDB82JSMC
+wzheng: UDB9TQ5P0
+chaoyang: UDE6JG87P
+shlao: UDWFCU0HY
+anusaxen: UE3J0MDL4
+minmli: UE5CJHNH3
+yuwan: UEE8X128Y
+xueli: UEEQ10T4L
+akostadi: UEEV72MFZ
+rpattath: UEGH1JZN1
+jfan: UEHSXN116
+skordas: UEL9SMWV8
+chuo: UFGRUV2B1
+scolange: UFH8W8U3F
+mgahagan: UFVMZJB6G
+pamoedom: UFZM9M3M5
+preichl: UGC8T8Y7Q
+bandrade: UH9M94G2D
+wduan: UML1DEFHC
+huirwang: UMTAZPTJA
+pmali: UNCR57Q6B
+xiyuan: UNF5B95K3
+tbuskey: UP2HXQSQ2
+yanyang: UQ56XHM89
+whu: UQJTB1W10
+kewang: URQL37FLM
+lwan: US1N51RT6
+knarra: USCCU4Z0W
+aiyengar: USUEN4PPH
+hongyli: USYJCHV08
+kuiwang: USYV0HTN2
+lzha: UU34YJLHE
+yunjiang: UUD6UUWN7
+jima: UUEJRQQRF
+kkulkarni: U0130D2J02G
+prubenda: U0130UWGREC
+cmeadors: U014T9UU8LU
+dbrahane: U01EUKLKGFJ
+asood: U01EUQ4PDRQ
+valiev: U01HXRWCW67
+rrasouli: U01J4GKLCLQ
+xzha: U01JSFYTD6J
+tsze: U014A42CGDN
+rbrattai: UNVQAGPGT
+sgao: UND36RY00
+ematysek: UJJTF6TK5
+dahmed: U019W3J1VEF
+weliang: UBWB79BNV
+kbharti: U01QZNYGWCX
+yli2: U01T26DP2G1
+qili: U01SBLKMZFZ
+Lena Horsley: U01SRL654UB
+xjiang: UM44FF4KG
+jitsingh: UMSJFPB37
+Ying Wang: U01T97U8S9K
+jianl: U01T9SZFKSS
+qili: U01SBLKMZFZ
+yli2: U01T26DP2G1
+arout: UGS4QRLN5
+nweinber: UUZRDBAF6
+jgeroge: U023N0T2EHH
+liqcui: U021ZE55KEZ
+jiwei: U027QMBNUTD
+jwei: U0245V3EC66
+aleskandro: U02AJU6BKT2
+Shivanthi: U029X7M6KGT
+Melvin Joseph: U029EJV606T
+Ying Zhang: U029UUDF2D8
+Abhishek Banerjee: U029EH4T6H5
+shudili: U02C0U93S9H
+liangli: U028EFFD8JU
+yingzhan: U029UUDF2D8
+jshu: U02CPDBQE3A
+jiezhao: U01UYTWLTPW
```

Comment:
Such user lists IMO are better off in some internal repo. Even path used `creds` suggests they are not good for public exposure.

Reply:
@akostadinov you mean in the `cucushift-internal` repo?  These are just Slack user alias and their corresponding user ID.  There's nothing secret about it, that's why I didn't put them into the internal repo to begin with.  There's a way to look up the users dynamically, but it's a slow process, which is why I went with a static mapping approach.

Reply:
Not only a matter of secrecy. It looks more like configuration for whatever will be scheduling this operation. Not something to belong to code repo.

Reply:
@akostadinov @liangxia ok, I'm changed the implementation and generate the user information dynamically instead of hardcoding users.  It's looking for everyone in the `openshift-qe` slack group.  PTAL.  Please note it's dependent on https://gitlab.cee.redhat.com/aosqe/cucushift-internal/-/merge_requests/9/

Reply:
@liangxia @akostadinov now that above PR is merged, this should work now.  PTAL and merge if there are no more issues.